src/drive/ducks/files/Container.jsx

```diff
@@ -71,7 +71,7 @@ const mapDispatchToProps = (dispatch, ownProps) => {
           displayCondition: selections =>
             hasWriteAccess && selections.length === 1
         },
-        availableOffline: {
+        'phone-download': {
```

Comment:
Not fan of using that kind of string as an object key in JS code, couldn' we stay with camelCase, even if it is ugly in Stylus files?